src/utils/response/responseHandlers.ts

```diff
@@ -1,58 +1,53 @@
 import { AxiosResponse } from 'axios';
-import { ApiResponse, ApiResponseOnSuccess } from './apiResponse';
+import { ApiResponse } from './apiResponse';
 
-class ResponseHandler {
-  async toApiResponse<T>(promise: Promise<AxiosResponse>) {
-    const dataPromise = await promise;
-    const apiResponse = (dataPromise.data as unknown) as ApiResponse<T>;
```

Comment:
Now that we have correct types coming in, I was able to remove this `as unknown as ApiResponse` cast

Reply:
@benj56 Please keep different changes separated in the PR. It is very hard to see why these changes were needed in the context of the story. If you like to improve something just create a new story and don't just implement it in the current branch.